<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraPostionDlg</class>
 <widget class="QDialog" name="CameraPostionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>传感器参数</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>视点</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_lon_s"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_lon">
       <property name="text">
        <string>经度：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_lat">
       <property name="text">
        <string>纬度：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_FanGunAngle"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_lat_s"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_hei"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_lat"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_lon"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_height">
       <property name="text">
        <string>高度：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_VerAngle"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_hei_s"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_horAngle"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_hor_s"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>水平方位角：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_ver_s"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>垂直俯仰角：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>翻滚角:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>焦距：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_Range"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_range_s"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Trace">
       <property name="text">
        <string>目标聚焦</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SetRotateAngle">
       <property name="text">
        <string>设置翻滚角</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_copy">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_set">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>301</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_FocusHei"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_Focus">
       <property name="text">
        <string>聚焦</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_FocusLat"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_FocusLon"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>经度</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>纬度</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>高度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>视场</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>水平视场角</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直视场角</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_VerVP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_HorVPCur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_VerVPCur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>度</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Target">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>目标</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>位置</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Lon">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>经度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>纬度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Hei">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>米</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Lat">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>高度：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>角度(欧拉角)</string>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x 轴：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>y 轴：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>z 轴：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AngleX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AngleY">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AngleZ">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>度</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设为新位置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>尾焰</string>
    </property>
    <widget class="QCheckBox" name="checkBox_Jet1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>尾焰1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Jet2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>尾焰2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Jet3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>尾焰3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Jet4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>尾焰4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loadJet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载尾焰</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>分辨率</string>
   </property>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直分辨率：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_HorRes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_VerRes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>水平分辨率：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Info">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Win">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_copy</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_set</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Set</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotSetNewPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Trace</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotTrace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Jet1</sender>
   <signal>clicked(bool)</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotJet1(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Jet2</sender>
   <signal>clicked(bool)</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotJet2(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Jet3</sender>
   <signal>clicked(bool)</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotJet3(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_loadJet</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotLoadJet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Jet4</sender>
   <signal>clicked(bool)</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotJet4(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_SetRotateAngle</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotSetRotateAngle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Focus</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Win</sender>
   <signal>clicked()</signal>
   <receiver>CameraPostionDlg</receiver>
   <slot>slotSetWin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCopy()</slot>
  <slot>slotSet()</slot>
  <slot>slotSetViewPort()</slot>
  <slot>slotSetNewPosition()</slot>
  <slot>slotTrace()</slot>
  <slot>slotTest()</slot>
  <slot>slotJet1(bool)</slot>
  <slot>slotJet2(bool)</slot>
  <slot>slotJet3(bool)</slot>
  <slot>slotJet4(bool)</slot>
  <slot>slotLoadJet()</slot>
  <slot>slotSetRotateAngle()</slot>
  <slot>slotSetWin()</slot>
  <slot>slotFocus()</slot>
 </slots>
</ui>
